### PR TITLE
Getallorders

### PR DESCRIPTION
**Pull request by Ivankivu**
What this PR does:-
* Gets a list of all orders
* Follow this link to see it action
https://fastfood-fast-api-heroku.herokuapp.com/

### CODE REVIEW

**Review by Kasulejoseph — COMMENTED** [review]
make use of the normal file structure.. you shouldn't be running the app directly from here
Also try as much as possible to implement OOP. at least create another file with a class for orders and be importing orders from their not directly

**Comment by Ivankivu** [comment]
Okay Thanks @Kasulejoseph let me make some changes

File: main/tests/test_orders.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from unittest import TestCase
+from views.orders import order_id,orderlist, OnlineRestuarant
+
+class OnlineRestuarantTest(unittest.TestCase):
+
+    def test_welcome(self):
+        self.assertTrue(True)
+
+    def test_get_orders(self):
+        if orderlist == []:
```
Comment (kaggwachristopher) [inline main/tests/test_orders.py:11]: I think this if statement should  be part of the code you are testing not part of the test...
So in otherwords just write a test to check whether  value error is actually raised..

**Review by kaggwachristopher — COMMENTED** [review]
just a little more to polish in the unittests on line 11.............
the other stuff seems ok to me,for now

File: main/tests/test_orders.py (hunk 1 above)
Comment (Ivankivu) [reply main/tests/test_orders.py:11]: @kaggwachristopher Thanx for the advise. let me apply it and see the results

**Pull request merged**